packages/grape-web-client/package.json

```diff
@@ -27,7 +27,7 @@
     "grape-browser": "^3.27.4",
     "grape-js-emoji": "^3.4.0",
     "grape-theme": "^3.6.0",
-    "grape-web": "^1.13.1",
+    "grape-web": "^1.13.1-19700.0.0",
```

Comment:
we could put "grape-web": "*" here since it's in the same repo now and versioned in git